src/Controller/index/LoginController.php

```diff
@@ -14,7 +22,7 @@ class LoginController extends Controller
 {
     public function loginPage(Request $request, AuthenticationUtils $authUtils, AuthorizationCheckerInterface $authCheck)
     {
-        if($this->isGranted('ROLE_ADMIN')) {
```

Comment:
to raczej nie jest związane z tytułem PR

Reply:
Oh, przez przypadek...